### ISSUE TITLE
Build fails with npm 7.x

### ISSUE THREAD

**Issue by ShutoAraki** [aws-solutions/machine-downtime-monitor-on-aws#2]
**Describe the bug**
When I execute `./build-s3-dist.sh`, I got the following error during dependency installation for filter-kinesis-stream lambda utils:

```
> filter-kinesis-stream@1.0.0 clean
> rm -rf node_modules/ dist/ coverage/ package-lock.json

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: filter-kinesis-stream@1.0.0
npm ERR! Found: react@16.14.0
npm ERR! node_modules/react
npm ERR!   peer react@"^16.8" from @react-native-community/async-storage@1.12.1
npm ERR!   node_modules/@react-native-community/async-storage
npm ERR!     peer @react-native-community/async-storage@"^1.11.0" from aws-appsync@4.1.1
npm ERR!     node_modules/aws-appsync
npm ERR!       aws-appsync@"^4.0.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"17.0.1" from react-native@0.64.2
npm ERR! node_modules/react-native
npm ERR!   peer react-native@">=0.59" from @react-native-community/async-storage@1.12.1
npm ERR!   node_modules/@react-native-community/async-storage
npm ERR!     peer @react-native-community/async-storage@"^1.11.0" from aws-appsync@4.1.1
npm ERR!     node_modules/aws-appsync
npm ERR!       aws-appsync@"^4.0.3" from the root project
```

**To Reproduce**
```
$ node -v
v14.8.0

$ npm -v
7.20.0
```

**Please complete the following information about the solution:**
- [ v1.0.0 ] Version: [e.g. v1.0.0]
- [ ap-northeast-1 ] Region: [e.g. us-east-1]
- [ No ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [ N/A ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses?
- [ N/A ] Were there any errors in the CloudWatch Logs?

**Comment by ShutoAraki** [comment]
This issue is solved when I downgraded `npm` to `6.14.14` following the advice from https://github.com/vouch-opensource/krell/issues/121

I suggest that you add "npm 6.x" to one of the prerequisites for customization.

**Comment by ericquinones** [comment]
Hi @ShutoAraki 

Thanks for letting us know. We'll make sure to note this in our `README`. To install using npm 7.x, you would need to run `npm install` with the `--legacy-peer-deps` flag.
